src/borg/archiver.py

```diff
@@ -2114,8 +2116,8 @@ def get_args(self, argv, cmd):
             if result.func != forced_result.func:
                 # someone is trying to execute a different borg subcommand, don't do that!
                 return forced_result
-            # the only thing we take from the forced "borg serve" ssh command is --restrict-to-path
```

Comment:
maybe don't kill the comment completely, but modify it to say:

we only take specific options from the forced "borg serve" command: